app/views/spree/admin/products/new.html.erb

```diff
@@ -66,7 +66,7 @@
       <div data-hook="new_product_shipping_category" class="col-12 col-md-4">
         <%= f.field_container :shipping_category do %>
           <%= f.label :shipping_category_id, Spree.t(:shipping_categories) %><span class="required">*</span>
-          <%= f.collection_select(:shipping_category_id, @shipping_categories, :id, :name, { include_blank: Spree.t('match_choices.none') }, { class: 'select2 w-100' }) %>
+          <%= f.collection_select(:shipping_category_id, @shipping_categories, :id, :name, { selected: @shipping_categories.first.id, include_blank: Spree.t('match_choices.none') }, { class: 'select2 w-100' }) %>
```

Comment:
```suggestion
          <%= f.collection_select(:shipping_category_id, @shipping_categories, :id, :name, { selected: @shipping_categories&.first&.id, include_blank: Spree.t('match_choices.none') }, { class: 'select2 w-100' }) %>
```

Reply:
@aleksandarpetrushev I would rather move setting these default values to controller